learing_journal/tests.py

```diff
@@ -42,23 +43,23 @@ def teardown():
 
 
 def test_model_gets_added(new_session):
-    assert len(new_session.query(MyModel).all()) == 0
-    model = MyModel(title="test", body='some text')
+    assert len(new_session.query(Journal).all()) == 0
+    model = Journal(title="test", body='some text')
```

Comment:
Your model's fields have changed since you first wrote this test. You now have two more fields: creation date and date last modified. They should also be a part of your model instance.

You should probably make a fixture function that generates some number of model instances as you need them. It makes doing things like this (and the next few tests) a little easier and less verbose.